boards/st/stm32n6570_dk/stm32n6570_dk_defconfig

```diff
@@ -0,0 +1,16 @@
+# SPDX-License-Identifier: Apache-2.0
+
+# Enable UART driver
+CONFIG_SERIAL=y
+
+# Enable GPIO
+CONFIG_GPIO=y
+
+# Console
+CONFIG_CONSOLE=y
+CONFIG_UART_CONSOLE=y
+
+# For now store the program in RAM during flashing
```

Comment:
Outdated. Aslo, should it be part of SoC config ?

Reply:
Keeping CONFIG_XIP=n. It gets overwritten if set in the SoC.
Removed CONFIG_FLASH_SIZE and CONFIG_FLASH_BASE_ADDRESS as they are automatically set to 0 when XIP=n